src/metadatas.rs

```diff
@@ -100,6 +102,21 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
             Some(Ok(s)) => !s.is_empty(),
             _ => false,
         },
+        lines_with_custom_color_count: gtfs
```

Comment:
lines_with_custom_color_count -> routes_with_custom_color_count?

Reply:
Ah, understood. Coherence > adhering to the GTFS spec.

Reply:
sorry for that :speak_no_evil: